build.sh

```diff
@@ -148,4 +148,12 @@ echo Generating $BUILD_TYPE makefiles
 cmake $CMAKE_GEN $CC_PREFIX $ICU_PATH -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
 
 $MAKE $MULTICORE_BUILD 2>&1 | tee build.log
+
+result=$?
+if [[ $result == 0 && ! -f ../ch ]]; then
+    ln -s BuildLinux/ch ../ch
```

Comment:
I assume you did this change to save some typing while testing manually? I tried it out and this broke my personal settings. I have "." at head of PATH. When I build at repo root and run "ch", it prints
```
dlopen() failed; dlerror says 'ch: cannot open shared object file: No such file or directory'
```

The same happens if I have `<repo>/BuildLinux` at head of PATH.

My workaround was a wrapper script "ch" in my PATH to invoke BuildLinux/ch. That doesn't have this problem.

So maybe a top-level symbolic link to "ch" isn't very useful?
- It has above problem. (maybe we can investigate and fix it.)
- The primary binary is "chakracore", not "ch". "ch" is mostly a testing binary. A top-level link feels over kill for it. 
- We might consider build type subdirs like BuildLinux/x64_debug, BuildLinux/x64_release. Top-level ch link doesn't work with that.